js/core/devices.d.ts

```diff
@@ -149,6 +149,14 @@ declare class DevicesObject {
      * @public
      */
     real(): Device;
+    /**
+     * @docid DevicesObjectMethods.isSimulator
+     * @publicName isSimulator()
+     * @return Boolean
+     * @prevFileNamespace DevExpress.core
+     * @hidden
+     */
+    isSimulator(): boolean;
```

Comment:
Why is doc comment added here?

Reply:
Where should I add they?

Reply:
Previously, the isSimulator method was internal. We want to publish it? Usually, we do not write doc comments for internal methods.

Reply:
It's marked as a "hidden" (https://github.com/DevExpress/DevExtreme/pull/13873/files#diff-5f66e6a1045e4cd65e0e6007dd944fe2R43) @IlyaKhD everything is good here?

Reply:
If the intention is to hide this member from wrappers and other generators, then doc-tags shouldn't be added at all.